package.json

```diff
@@ -59,7 +59,7 @@
   },
   "pnpm": {
     "overrides": {
-      "@rspack/core": "npm:@rspack/core-canary@1.0.0-canary-876f80b-20240827032943"
+      "@rspack/core": "npm:@rspack/core-canary@1.0.0-canary-fca07da-20240704070545"
```

Comment:
how to abandon this being updated

Reply:
We could add `@rspack/core` and `@rspack/core-canary` to https://github.com/web-infra-dev/rslib/blob/main/.github/renovate.json5#L50-L56.